### PR TITLE
[Integration][FireHydrant] Added _target='blank' attribute to spec links

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description
Update the description content and the attribute target="_blank" to anchor tags in descriptions to open a new browser tab instead of the current browser.

- [ ] Documentation (added/updated documentation)

### CODE REVIEW

File: integrations/firehydrant/CHANGELOG.md [hunk 1]
```diff
@@ -13,416 +13,358 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Upgraded integration dependencies (#1)
 
-
 # Port_Ocean 0.1.58 (2024-07-31)
 
 ### Improvements
 
+- Added _target='blank' attribute to spec links to open a new browser tab instead of the current browser. (#58)
```
Comment (Tankilevitch) [inline integrations/firehydrant/CHANGELOG.md:20]: wrong version in changelog, also why removed the space separating from each version it supposed to be two lines

File: integrations/firehydrant/pyproject.toml
Comment (Tankilevitch) [inline integrations/firehydrant/pyproject.toml:1]: missing version bump

File: integrations/firehydrant/.port/spec.yaml [hunk 2]
```diff
@@ -14,12 +14,12 @@ configurations:
     required: true
     type: string
     sensitive: true
-    description: API token for your FireHydrant account, for example, "jakdlfja933948anz.cnapaira". It can be found on your <a href="https://app.firehydrant.io/organizations/bots">Firehydrant dashboard</a>
+    description: API token for your FireHydrant account, for example, "jakdlfja933948anz.cnapaira". It can be found on your <a target="_blank" href="https://app.firehydrant.io/organizations/bots">Firehydrant dashboard</a>
```
Comment (PeyGis) [inline integrations/firehydrant/.port/spec.yaml:17]:
```suggestion
    description: The API token for your FireHydrant account used for authentication. To create a token, see the  <a target="_blank" href="https://docs.firehydrant.com/docs/api-keys">FireHydrant document</a>
```

File: integrations/firehydrant/.port/spec.yaml [hunk 3]
```diff
@@ -14,12 +14,12 @@ configurations:
...
   - name: apiUrl
     type: string
     required: true
     default: https://api.firehydrant.io
-    description: "FireHydrant API URL, default is `https://api.firehydrant.io`"
+    description: FireHydrant API URL, (e.g., "https://api.firehydrant.io"). To identify your base URL, see the <a target="_blank" href="https://developers.firehydrant.com/#/</a>
```
Comment (PeyGis) [inline integrations/firehydrant/.port/spec.yaml:22]:
```suggestion
    description: The FireHydrant API URL. If not provided, the default value is https://api.firehydrant.io
```

**Review by PeyGis — CHANGES_REQUESTED** [review]

**Review by PeyGis — APPROVED** [review]

**Review by Tankilevitch — APPROVED** [review]
LGTM

**Pull request merged**